src/App.js

```diff
@@ -18,9 +17,11 @@ function App() {
           <AddItem />
         </Route>
       </Switch>
-      <h1>Shopping List</h1>
-      <ItemsList />
-      <AddItemsToList />
+      {localStorage.getItem('newToken') ? (
+        <Redirect to="/list" />
+      ) : (
+        <CreateList />
```

Comment:
Thank you Luka for this suggestion. We actually had it that way in the beginning, but since the task doesn't say that the nav shouldn't be visible anymore we changed it back (thinking, this would be a AC in another task). Okay then, we will change it like in the wireframe.

Reply:
After changing my browser, I realized it wasn't only my devtools in firebox but really that way (your first comment). Will do all changes immediately 👍

Reply:
Yep, you are totally on a right track :). The main idea is to prevent users from accessing `Add Item` and `List view` if they don't have a token saved in the localStorage. Therefore, the only visible page could be a welcome page that renders the `Create List` button.

Reply:
Great this makes sense- thanks Luka for clarifying!